### PR TITLE
tdusk v1.18 update

### PR DESCRIPTION
**Pull request by mrhoribu**
1.18 (2020-08-16) - Slight update for attacksript/bigshot users to not get stuck and better optimize round checking/finish for them. Change GameObj.npcs references to GameObj.targets

### CODE REVIEW

File: scripts/tdusk.lic [hunk 1]
```diff
@@ -230,31 +235,6 @@ def attack(number)
 				end
 			end
 
-		#Bard Routine for Jovalon
```
Comment (sandersch) [inline scripts/tdusk.lic:233]: I feel attacked

File: scripts/tdusk.lic [hunk 2]
```diff
@@ -181,11 +186,11 @@ def attack(number)
 				end
 				pause 0.1
 			end
-			Script.start("foreach", "in room; look item") if UserVars.tdusk[:wave_number] != 25
-			Script.start("foreach", "in room; get item; stow item") if UserVars.tdusk[:wave_number] == 25
+			Script.start("foreach", "n=/^((?!arrow|pill).)*$/ in room; look item") if UserVars.tdusk[:wave_number] != 25 && !GameObj.loot.nil?
+			Script.start("foreach", "n=/^((?!arrow|pill).)*$/ in room; get item; stow item") if UserVars.tdusk[:wave_number] == 25 && !GameObj.loot.nil?
 
 		#Sorcerer Routine for Rasko
-		elsif !(npc.status =~ /dead|gone/) && Char.name =~ /Rasko|Roiken|Hammibal|Ushakaron/
```
Comment (sandersch) [inline scripts/tdusk.lic:188]: Beldannon will never financially recover from this

File: scripts/tdusk.lic (hunk 1 above)
Comment (mrhoribu) [reply scripts/tdusk.lic]: Ooops

File: scripts/tdusk.lic (hunk 2 above)
Comment (mrhoribu) [reply scripts/tdusk.lic:188]: Added back in.

**Pull request merged**